internal/build/container_resolver.go

```diff
@@ -35,26 +68,29 @@ func (r *ContainerResolver) ContainerIDForPod(ctx context.Context, podName k8s.P
 	}
 
 	if len(containers) == 0 {
-		return "", fmt.Errorf("no containers found with name %s", podName)
-	}
-
-	// On GKE, we expect there to be one real match and one spurious match -- a
-	// container running "/pause" (see: http://bit.ly/2BVtBXB); filter it out.
-	if len(containers) > 2 {
-		var ids []string
-		for _, c := range containers {
-			ids = append(ids, k8s.ContainerID(c.ID).ShortStr())
+		return "", containerNotFound{
+			Message: fmt.Sprintf("no containers found in pod: %s", podName),
```

Comment:
nit: no MATCHING containers found in pod

Reply:
done